contributors/design-proposals/scheduling/hugepages.md

```diff
@@ -0,0 +1,308 @@
+# HugePages support in Kubernetes
```

Comment:
Weird that this didn't get treated as a move

Reply:
It was moved to resource-management. This is a dupe.